Reports/router.js

```diff
@@ -1,4 +1,38 @@
 var express = require('express');
+var mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost:27017');
+var reportModels = require('./reportModels')
+var EventReport = reportModels.EventReport;
+var OtherReport = reportModels.OtherReport;
 var reportRouter = express.Router();
 
-module.exports = { reportRouter };
\ No newline at end of file
+reportRouter.post('/', (req, res) => {
+    var eventReport = new EventReport({
+        type: 'event',
+        approvalStatus: 'notApproved'
+    });
+    
+    eventReport.save(function(err, savedReport) {
+        if (err)
+            res.send(err);
+        
+        console.log({
+            message: 'report created',
+            id: savedReport.id
+        });
+        res.status(201).location(savedReport.id).end();
+    });
+    
+});
+
+reportRouter.get('/', (req, res) => {
+    var Report = mongoose.model('EventReport');
```

Comment:
I originally did it this way because that's what the example code showed, but we can actually just use the ```EventReport``` object obtained from reportModels.